services/checks/checks.go

```diff
@@ -274,16 +301,23 @@ func (s *Service) executeChecks(ctx context.Context) {
 
 // executeMySQLChecks runs MySQL checks for available MySQL services.
 func (s *Service) executeMySQLChecks(ctx context.Context) ([]string, error) {
-	targets, err := s.findTargets(models.MySQLServiceType)
-	if err != nil {
-		return nil, errors.Wrap(err, "failed to find proper agents and services")
-	}
-
 	checks := s.getMySQLChecks()
 
 	var res []string
-	for _, target := range targets {
-		for _, c := range checks {
+	for _, c := range checks {
+		minAgentVersion, err := getMinAgentVersionForCheckType(c.Type)
+		if err != nil {
+			s.l.Errorf("failed to get min agents version for type %s, reason: %s", c.Type, err)
```

Comment:
In general, we use warnings for handled errors (that are not visible to users). Please do that everywhere.

Reply:
Also, the error message would be really bad:

> failed to get min agents version for type MONGODB_GETCMDLINEOPTS, reason: missing min agent version for check type: MONGODB_GETCMDLINEOPTS

It should not repeat itself and be properly formatted full sentence. Please do that everywhere.